src/main/java/io/pivotal/cla/data/repository/IndividualSignatureRepository.java

```diff
@@ -35,7 +35,7 @@ default IndividualSignature findSignaturesFor(User user,  String claName) {
 		return results.isEmpty() ? null : results.get(0);
 	}
 
-	@Query("select s from IndividualSignature s where (s.cla.name = :#{#claName} or s.cla.name in (select distinct c.supersedingCla.name from ContributorLicenseAgreement c where c.name = :#{#claName})) and (s.githubLogin = :#{#u.githubLogin} or s.email in (:#{#u.emails}))")
+	@Query("select s from IndividualSignature s where (s.cla.name = :#{#claName} or s.cla.name in (select distinct c.supersedingCla.name from ContributorLicenseAgreement c where c.name = :#{#claName})) and (s.githubLogin = :#{#u.githubLogin} or s.email in (:#{#u.emails.empty ? '' : #u.emails}))")
```

Comment:
The only way I'm aware of dealing this within the database is to use a trigger. Do you have alternative ideas?